app/src/main/java/org/simlar/service/liblinphone/LinphoneThread.java

```diff
@@ -372,7 +549,17 @@ public void callState(final LinphoneCore lc, final LinphoneCall call, final Linp
 
 			final String number = getNumber(call);
 			final LinphoneCall.State fixedState = fixLinphoneCallState(state);
-			Lg.i("callState changed state=", fixedState, " number=", new CallLogger(call), " message=", message);
+			final VideoState videoState = createVideoState(fixedState, call);
+
+			updateVideoState(videoState);
+
+			Lg.i("callState changed state=", fixedState, " number=", new CallLogger(call), " message=", message, " videoState=", videoState);
+
+			if (videoState == VideoState.REMOTE_REQUESTED) {
+				Lg.i("remote requested video");
+				/// NOTE: this needs to happen directly, posting to linphone thread might take to log
```

Comment:
`... to long`?

Reply:
0bad01db1b5b807e08678980659154b7b736f6a4